index.html

```diff
@@ -1063,41 +1068,26 @@ <h4>serializeDerivedProofValue</h4>
                     </li>
                     <li>
 Initialize |components| to an array with elements containing the values of
-|bbsProof|, |compressedLabelMap|, |mandatoryIndexes|, |selectiveIndexes|, and
-|presentationHeader|.
-                    </li>
-                  </ol>
-                </li>
-                <li>
-If |featureOption| equals `"pseudonym_issuer_pid"`:
-                  <ol class="algorithm">
-                    <li>
-Initialize |proofValue| to start with the
-disclosure proof header bytes `0xd9`, `0x5d`, and `0x07`.
-                    </li>
-                    <li>
-Initialize |components| to an array with elements containing the values of
 |bbsProof|, |compressedLabelMap|, |mandatoryIndexes|, |selectiveIndexes|,
-|presentationHeader|, and |pseudonym|.
+|presentationHeader| and |L|.
```

Comment:
I see this changed from `pseudonym` to `L` ... I'm not checking this closely right now, but will do so during implementation.

Reply:
@dlongley both *anonymous holder binding* and *pseudonyms* features need the *L* parameter now.  Pseudonyms still need *pseudonyms* ;-).